assets/scripts/lobby9.js

```diff
@@ -713,9 +712,13 @@ function drawAndPositionAvatars() {
   function drawGuns() {
     $(".gun img").css("width", $("#mainRoomBox div").width() + "px"); 
     $(".gun").css("width", $("#mainRoomBox div").width() + "px"); 
-
-    
-
+    if (docCookies.hasItem('optionProposedTeamIcon') === true){
+        if (docCookies.getItem('optionProposedTeamIcon') === true) {
+            updateGunImage("shield");
+        }
+    } else {
+        updateGunImage("gun");
```

Comment:
Shouldn't need to do this every time drawGuns() is called. Should only change the src attribute on two occasions, 1) On load, 2) When the user clicks the option (i.e. only need to set the src attribute in the initialiseEventListener area)

Reply:
Accepted. Will be resolved in the next commit